src/qos_p256/fuzz/fuzz_targets/9_decrypt_shared_secret.rs

```diff
@@ -0,0 +1,31 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+
+// use qos_p256::encrypt::P256EncryptPair;
+use qos_p256::encrypt::P256EncryptPublic;
+
+#[derive(Clone, Debug, arbitrary::Arbitrary)]
+pub struct FuzzKeyDataStruct {
+	public: Box<[u8]>, // this should be 65 byte in theory, but both 33 and 65 byte work
```

Comment:
Since https://github.com/tkhq/qos/pull/499 was merged we don't accept 33 bytes inputs anymore. I think you can remove or amend this comment.